lms/djangoapps/teams/static/teams/js/views/edit_team.js

```diff
@@ -21,24 +21,21 @@
 
                 initialize: function(options) {
                     this.teamEvents = options.teamEvents;
-                    this.courseID = options.teamParams.courseID;
-                    this.topicID = options.teamParams.topicID;
+                    this.context = options.context;
+                    this.topic = options.topic;
```

Comment:
The context is what is passed down from the server to the factory class. I'm trying to treat is as immutable so any dynamic values get passed in separately.